graphein/protein/visualisation.py

```diff
@@ -23,6 +24,8 @@ def plot_pointcloud(mesh: Meshes, title: str = "") -> None:
     :param title: Title of plot
     :return:
     """
+    # TODO: Meshes needs to be imported
```

Comment:
Noted, will take care of

Reply:
Actually, does it? I thought `from __future__ import annotations` would avoid having to import `Meshes`

Reply:
I see. Went online and read about the `from __future__ import annotations` piece. It's intriguing, though I think doing so might mean we're not being "explicit" about imports and types. You might be able to reduce future maintenance mental burden by sticking to "every name must be explicitly imported or declared" as a habit throughout the codebase.

Reply:
One nice thing I came across is that it’s avoided some circular dependency problems when certain things are imported just for type hinting. I’m not this if this is bad practice & covering up something a little more sinister (i.e. a minor problem I’ve engineered through the non-empty `__init__.py`s).